DangoPlop/Assets/Scripts/BulletController.cs

```diff
@@ -7,7 +7,8 @@ public class BulletController : MonoBehaviour {
 	public Vector2 Speed;
 
 	Rigidbody2D rb;
-
+	private GameObject PlayerControl;
+	private PlayerController ammo;
```

Comment:
i don't understand the oo (object oriented) design here. Why does the bullet have access to the player? The bullet is just a bullet. What you're doing here is basically saying, "the bullet contains the player controller". Naming the variable ammo and the class being playercontroller is also kind of weird.